deploy/packages/bbp-packages.yaml

```diff
@@ -116,11 +116,6 @@ packages:
       - mpi
       - python
     specs:
-      - neurodamus@plasticity
-      - neurodamus@plasticity+coreneuron^coreneuron+knl
-      - neurodamus@master~coreneuron
-      - neurodamus@hippocampus~coreneuron
-      - neurodamus+coreneuron^coreneuron+knl
       - neurodamus-neocortex
       - neurodamus-hippocampus
       - neurodamus-thalamus
```

Comment:
additional `+knl`  are needed

Reply:
see the whole file. There are +coreneuron+knl versions

Reply:
Ahh, now I see that! all good!